kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -388,14 +388,17 @@ define([
             });
 
             const layout = orderedParams.map(function (parameterId) {
-                let id = html.genId();
+                const elementId = html.genId();
+                const advanced = paramMap[parameterId].ui.advanced ? parameterId : false;       
```

Comment:
it really was this simple in the end tho

Reply:
Simple solution are the best 🎉